### PR TITLE
fix: design discrepancies and feedback for OP. My profile

### PR DESCRIPTION
**Pull request by bastantebom**
**Ticket:** [Bugs related to: OP. My profile: about tab (excluding all other tabs)](https://www.notion.so/socious/Bugs-related-to-OP-My-profile-about-tab-excluding-all-other-tabs-712b115117c04ba585626d26440e5403?pvs=4)

**Definition of done:**
- [x]  1. Figma includes data related to the “industry” and “size” - It was already implemented we just don't have data
- [x]  2. The website link should open in the new tab
- [x]  3. The hiring badge is not aligned on top on the same level as the name (please check Figma)(Minh feedback on GitHub)
- [x]  4. Share button missing spacing between icon and text(Minh feedback on GitHub)
- [x]  5. border missing for country flag(Minh feedback on GitHub)
- [x]  6. The cursor pointer missing on the "see more" hover(Minh feedback on GitHub)

**Media:**

https://github.com/socious-io/web-app-v2/assets/5590282/da62f96d-b2fe-4eb9-82ee-e164f59399af

### CODE REVIEW

File: src/Nowruz/modules/userProfile/components/profileHeader/desktopHeader.tsx [hunk 1]
```diff
@@ -79,7 +82,11 @@ export const DesktopHeader: React.FC<DesktopHeaderProps> = ({
         )}
         {!myProfile && (
           <div className={`${css.actionDiv} right-8 w-fit`}>
-            <Button color="primary" variant="outlined" style={{ flex: '1', height: '40px', fontSize: '14px' }}>
+            <Button
+              color="primary"
+              variant="outlined"
+              style={{ flex: '1', height: '40px', fontSize: '14px', gap: '6px' }}
+            >
```
Comment (marjan-ara) [inline src/Nowruz/modules/userProfile/components/profileHeader/desktopHeader.tsx:89]: Could you please replace inline style with customStyle and pass a className to this button? I think it has been my mistake

**Review by marjan-ara — CHANGES_REQUESTED** [review]

**Review by marjan-ara — APPROVED** [review]

**Comment by aziinz** [comment]
@bastantebom The 
Size
Size data is wrong

**Comment by bastantebom** [comment]
> @bastantebom The Size Size data is wrong

I will check the current data.

**Comment by jeyem** [comment]
> > @bastantebom The Size Size data is wrong
> 
> @aziinz I will check the current data.

@bastantebom we have constant for this please check constant file to find size mapping

**Comment by bastantebom** [comment]
> > > @bastantebom The Size Size data is wrong
> > 
> > 
> > @aziinz I will check the current data.
> 
> @bastantebom we have constant for this please check constant file to find size mapping

@jeyem can you point me to the correct file where I can see this mapping, I couldn't find any relevant file. Thanks

**Pull request merged**